README.md

```diff
@@ -70,3 +70,73 @@ The newest version of freeplane (1.11.10) is needed.
 **There are no learning objectives or tests for an overview branch. The overview branch lets trainers pick and choose relevant topics within a branch to teach at any level required so it doesn’t need to complete the learning objectives that are strictly specified in the leaf sections.** 
 
 **In Overview, only the Header and Description are required.**
+
+## How to TAG using the skill tree
+
+Before showing examples, we are reiterating important rules and definitions of skills and the skill tree.
+All the rules can be found in the skill tree [Readme.md](https://github.com/HPC-certification-forum/skill-tree/blob/master/README.md).
+
+### Skills
+
+A skill is a learnable unit of content, which can be taught in no more then 4 hours and no less then 1 hours.
+A skill does not have a difficulty but can have requirements.
+
+If the content of a skill has more content then or grows beyond 4 hours of teaching, it will be split into sub skills.
+The simple, original name remains and becomes an overview.
```

Comment:
The simple, original name remains and becomes an overview item.

or "... overview entry in the skill tree."?

Reply:
Yep, made it:
overview leaf in the skill tree